packages/xstate-graph/test/graph.test.ts

```diff
@@ -432,3 +571,102 @@ describe('@xstate/graph', () => {
     });
   });
 });
+
+it('simple paths for reducers', () => {
+  const a = traverseShortestPlans(
+    {
+      transition: (s, e) => {
+        if (e.type === 'a') {
+          return 1;
+        }
+        if (e.type === 'b' && s === 1) {
+          return 2;
+        }
+        if (e.type === 'reset') {
+          return 0;
+        }
+        return s;
+      },
+      initialState: 0 as number
```

Comment:
nit: 0 is already a number 😅 
```suggestion
      initialState: 0
```